.github/actions/admin/action.yml

```diff
@@ -79,7 +82,8 @@ runs:
         protect_lockfile: '${{ inputs.protect_lockfile }}'
         terraform_wrapper: false
 
-    - name: 'Init'
+    - name: 'Terraform Init'
+      if: ${{ inputs.terraform_init == 'true' }}
```

Comment:
Nit: should we check the literal string "true"? Or something truthy?

Reply:
Sigh, you cant set a type for composite actions, so I believe everything is a string. I think checking for string == 'true' is ok as that is the default. Open to suggestions...

Maybe we could change to `prevent_init` and default it to false as I feel the frequent case is using enabling init.